### ISSUE TITLE
pivot[owner][first_name] has issue

### ISSUE THREAD

**Issue by bkrajendra** [daftspunk/oc-crm-plugin#4]
pivot[owner][first_name]  in column definition for opportunityofferingpivot model does not work properly.
If pivot[owner][first_name] kept there is shows SQL error.

``` sql
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'pivot_owner' in 'order clause' (SQL: select `acme_crm_offerings`.*, `acme_crm_opportunities_offerings`.`opportunity_id` as `pivot_opportunity_id`, `acme_crm_opportunities_offerings`.`offering_id` as `pivot_offering_id`, `acme_crm_opportunities_offerings`.`price` as `pivot_price`, `acme_crm_opportunities_offerings`.`cost` as `pivot_cost`, `acme_crm_opportunities_offerings`.`owner_id` as `pivot_owner_id` from `acme_crm_offerings` inner join `acme_crm_opportunities_offerings` on `acme_crm_offerings`.`id` = `acme_crm_opportunities_offerings`.`offering_id` where `acme_crm_opportunities_offerings`.`opportunity_id` = 1 order by `pivot_owner` desc)
```

while of pivot[owner][first_name] is replaced by pivot[owner], it displays entire user json as:

``` json
{"id":1,"first_name":"Admin","last_name":"Person","login":"admin","email":"admin@domain.tld","permissions":"","is_activated":true,"activated_at":null,"last_login":"2016-05-23 11:01:56","created_at":"2016-05-23 11:01:37","updated_at":"2016-05-23 11:01:56","is_superuser":1}
```

Not getting how to resolve this. whats needed is to display only owner name here as displayed in Notes list.

**Comment by pbsmultimedia** [comment]
@bkrajendra have you manage to solve this?

**Comment by bkrajendra** [comment]
@pbsmultimedia not yet! Little bzy in some other project. But this plugin is good starting point for a CRM project, I do need proper solution. let me know if you get it resolved.

**Comment by pbsmultimedia** [comment]
@bkrajendra i got there with help of slowpokefarm on october slack

file: OpportunityOfferingPivot.php

...
    public function getOwnerNameAttribute(){
        return $this->owner ? $this->owner->first_name : null;
    }    

and the respective fields.yaml:

....

```
pivot[owner_name]:
    label: Owner
    type: relation
    nameFrom: first_name
    sortable: false
```

He says that this is not a good practise, but it works.

**Comment by BlazL** [comment]
This works fine.

**Comment by daftspunk** [comment]
Alternative fix could also be: 

```
    first_name:
        valueFrom: pivot[owner][first_name]
        label: Assigned to
        width: 250px
```

However, this will be fixed in Build 380+ (#3)

Refs: https://github.com/octobercms/october/commit/46c473e2f214c3b4645849f46bd951931f8a7810
